Makefile

```diff
@@ -2,7 +2,7 @@ SPEC_DIR = ./specs
 SSZ_DIR = ./ssz
 TEST_LIBS_DIR = ./tests/core
 TEST_GENERATORS_DIR = ./tests/generators
-PY_SPEC_DIR = $(TEST_LIBS_DIR)/pyspec
+PY_SPEC_DIR = $(TEST_LIBS_DIR)/pyspec/eth2spec
```

Comment:
The outputs of the coverage report, test report, caches, etc. go into this directory I think. The `eth2spec` dir is a python *package*, so I think it shouldn't contain any such test output data.